packages/department-list/src/sass/_module.scss

```diff
@@ -0,0 +1,35 @@
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// department-list module
+// A navigation list style for listing many departments.
+//
+// Content:
+// - Department list style
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+
+@import '_globals.scss';
+
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// Contents links style
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+.uikit-department-list {
+
+	.uikit-body & > li {
```

Comment:
@dominikwilkowski Sure? I think I do, to unset styles that are being otherwise inherited from `link-lists` (the margins)?